src/apps/tools.clj

```diff
@@ -54,6 +54,14 @@
               :tool_request_status)
       remove-nil-vals))
 
+(defn- format-container-settings
+  [container-settings include-defaults]
+  (if include-defaults
+      (merge {:max_cpu_cores (config/default-cpu-limit)
+              :memory_limit (config/default-memory-limit)}
+             container-settings)
```

Comment:
`merge` overwrites values from the first map with ones from the second, so this should suffice, I think

Reply:
I think that should suffice as well. 👍